src/Button/Button.js

```diff
@@ -56,9 +56,9 @@ export const Button = ({ className, children, onClick, ref, variant, size, type,
   // Is disabled
   if (isDisabled) {
     levers.disabled = classnames('su-bg-black-20 su-text-black su-border-2 su-border-black-20 su-border-solid su-pointer-events-none')
+    levers.variant = classnames(levers.variant, 'su-border-none', {'su-bg-digital-red': false, 'su-border-solid': false, 'su-text-digital-red': false, 'su-text-white': false})
```

Comment:
Heads up I made some tweaks to this one to keep the borders, but I'm merging everything in and will change this line back. Thanks for adding all the test stuff!